dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/newsession/NewSessionModel.java

```diff
@@ -102,64 +118,60 @@ public StringProperty proxyServerProperty() {
     }
 
     public Session toSession() {
-
         ALERT.setHeaderText(null);
         Ds3Client client = null;
         try {
-
             final Stage stage = (Stage) ALERT.getDialogPane().getScene().getWindow();
-            stage.getIcons().add(new Image(ImageURLs.DEEPSTORAGEBROWSER));
-
+            stage.getIcons().add(new Image(ImageURLs.DEEP_STORAGE_BROWSER));
             if (this.getProxyServer() != null && this.getProxyServer().equals("")) {
                 this.setProxyServer(null);
             }
             client = Ds3ClientBuilder
-                    .create(this.getEndpoint() + ":" + this.getPortNo(),
+                    .create(this.getEndpoint().trim() + ":" + this.getPortNo().trim(),
                             new Credentials(this.getAccessKey(),
                                     this.getSecretKey()))
                     .withHttps(false).withProxy(this.getProxyServer())
                     .build();
-
-            final GetSystemInformationSpectraS3Response sysreponse = client.getSystemInformationSpectraS3(new GetSystemInformationSpectraS3Request());
-            final GetServiceResponse response = client.getService(new GetServiceRequest());
-            return new Session(this.getSessionName(), this.getEndpoint(), this.getPortNo(), this.getProxyServer(), client);
-
-
+            client.getService(new GetServiceRequest());
+            return new Session(this.getSessionName(), this.getEndpoint(), this.getPortNo(), this.getProxyServer(), client, this.getDefaultSession());
         } catch (final UnknownHostException e) {
+            LOG.error("Invalid Endpoint Server Name or IP Address", e);
             ALERT.setTitle("Invalid Endpoint");
             ALERT.setContentText("Invalid Endpoint Server Name or IP Address");
             ALERT.showAndWait();
-
         } catch (final FailedRequestUsingMgmtPortException e) {
+            LOG.error("Attempted data access on management port -- check endpoint", e);
             ALERT.setContentText("Attempted data access on management port -- check endpoint");
             ALERT.showAndWait();
         } catch (final FailedRequestException e) {
             if (e.getStatusCode() == 403) {
                 if (e.getError().getCode().equals("RequestTimeTooSkewed")) {
+                    LOG.error("Failed To authenticate session : Client's clock is not synchronized with server's " +
```

Comment:
In the event that the clock is skewed, it looks like this will display two errors one right after the other.  Consider putting the second authentication error into an else block